src/components/Card/LandingPageCard/LandingPageCard.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import propTypes from 'prop-types';
+import { motion } from 'framer-motion';
+
 const LandingPageCard = ({ image, title }) => {
   return (
-    <>
-      <div className=" max-h-lg max-w-sm sm:w-1/2 lg:w-1/4  overflow-hidden shadow-lg transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-105 rounded">
-        <img className="w-full" src={image} alt="Train to Maintain landing" />
-        <div className="font-bold text-xl text-purple-600 text-center p-10">
-          {title}
-        </div>
+    <motion.div
+      whileHover={{ scale: 1.02 }}
+      className=" max-h-lg max-w-sm sm:w-1/2 lg:w-1/4 overflow-hidden shadow-2xl  "
```

Comment:
dimension still not correct in small or even large it looks a square more than a rectangular, I suggest you make it fixed PX